src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -214,7 +212,9 @@ private static void ProcessRequest(HttpWebRequest request)
             }
 
             var activity = WebRequestActivitySource.StartActivity(ActivityName, ActivityKind.Client);
+            var activityContext = Activity.Current?.Context ?? default;
 
+            InstrumentRequest(request, activityContext);
```

Comment:
AddRequestTagsAndInstrumentRequest() down the line will call InstrumentRequest Again, so need to chaneg that too

Reply:
I removed from that part. So, now we just have one call